### PR TITLE
Triggers for company

### PR DESCRIPTION
**Pull request by msgIasmina**
triggers for company and user company validation

### CODE REVIEW

File: src/test/java/msg/flight/manager/controllers/user/auth/CUDUserControllerTest.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ public void updateUserData_failsWith403_whenOtherCompanyManager() throws Excepti
 
     @Test
     public void updateUserData_succeedsWith200_whenAdminUser() throws Exception {
```
Comment (LeonteDenismsgDev) [inline src/test/java/msg/flight/manager/controllers/user/auth/CUDUserControllerTest.java:153]: ai adaugat functia cretea (create TYPO) DBUser? Ca o tot caut si no o gasesc

File: src/test/java/msg/flight/manager/controllers/user/auth/SpringControllerTest.java [hunk 2]
```diff
@@ -54,6 +61,8 @@ public void setup() throws Exception {
     @After
     public void clearSetup() {
         userRepository.deleteUser(testUser.getUsername());
+        companyRepository.remove("Test");
+         companyRepository.remove("OtherTest");
         tokenRepository.deleteToken(token);
     }
```
Comment (LeonteDenismsgDev) [inline src/test/java/msg/flight/manager/controllers/user/auth/SpringControllerTest.java:67]: e typo la cretea

**Review by LeonteDenismsgDev — APPROVED** [review]
pls fix typos, when fixed push without my approval

File: src/test/java/msg/flight/manager/controllers/user/auth/SpringControllerTest.java (hunk 2 above)
Comment (msgIasmina) [reply src/test/java/msg/flight/manager/controllers/user/auth/SpringControllerTest.java:67]: la ce cretea te referi?

Comment (LeonteDenismsgDev) [reply src/test/java/msg/flight/manager/controllers/user/auth/SpringControllerTest.java:68]: da

Comment (LeonteDenismsgDev) [reply src/test/java/msg/flight/manager/controllers/user/auth/SpringControllerTest.java:67]: la functia de mai jos, la numele ei

**Pull request merged**